build/Dockerfile

```diff
@@ -22,10 +22,18 @@ ARG BASE_URL="https://archive.apache.org/dist/maven/maven-3/${MAVEN_VERSION}/bin
 
 USER 0
 
+RUN mkdir -p ${MAVEN_HOME} \
+    && curl -Lso /tmp/maven.tar.gz ${BASE_URL}/apache-maven-${MAVEN_VERSION}-bin.tar.gz \
```

Comment:
We should look to using that version with Mvnd as well, that is removing the embedded one from the Maven Daemon.